packages/bridge-controller/package.json

```diff
@@ -73,9 +73,9 @@
     "typescript": "~5.2.2"
   },
   "peerDependencies": {
-    "@metamask/accounts-controller": "^23.0.0",
+    "@metamask/accounts-controller": "^24.0.0",
```

Comment:
Accounts controller bump will not affect bridge controller

Reply:
Nit: can we add least changelog entry for this bump under unreleased section

Reply:
Definitely! Lemme add that.

Reply:
f3ef047603e57cf66102399cdc303369c4e6eb08